README.md

```diff
@@ -14,10 +12,20 @@ slightly stricter schema version.
 
 Please review [fonto.json](./fonto.json) for the schema locations correlating with each shell.
 
-## To use
+## Fonto Editor schema compilation
 
-- Create a ZIP archive for the contents of this repository.
+This repository can be used to create schema packages required for the Fonto Editor to function.
 
-- Follow the instructions to [integrate a schema](
-  http://documentation.fontoxml.com/editor/latest/integrate-a-schema-3099089.html) in your FontoXML editor as per
-  normal.
+Follow the instructions on [How to compile an updated schema into an existing editor](https://documentation.fontoxml.com/latest/re-compile-the-schema-1fe416604f65).
+
+## DITA Open Toolkit plugin
+
+This repository doubles as a [DITA Open Toolkit](https://www.dita-ot.org/download) plugin.
+
+- Install the plugin using `dita install` ([documentation](https://www.dita-ot.org/dev/topics/plugins-installing.html)).
```

Comment:
NIT: should be an ordered lists. Otherwise the steps may be executed in arbitrary order which is not the case.

Reply:
Updated the unordered list to an ordered list.